packages/material-ui/src/Backdrop/Backdrop.js

```diff
@@ -56,9 +56,11 @@ const BackdropRoot = experimentalStyled(
 const Backdrop = React.forwardRef(function Backdrop(inProps, ref) {
   const props = useThemeProps({ props: inProps, name: 'MuiBackdrop' });
   const {
+    components = {},
+    componentsProps = {},
     children,
     className,
-    invisible = false,
+    invisible: invisibleProp = false,
```

Comment:
This looks unnecessary to me. We don't declare `invisible` in the function body, no?

Reply:
Yes, probably you are right used Badge as a template and just named it can fix it a sec.